docs/book/src/clusterctl/provider-contract.md

```diff
@@ -317,12 +317,20 @@ Templates writers should use the common variables to ensure consistency across p
 | ---------------------- | ----------------- | ------------------------------------------- |
 |`--target-namespace`| `${NAMESPACE}` | The namespace where the workload cluster should be deployed |
 |`--kubernetes-version`| `${KUBERNETES_VERSION}` | The Kubernetes version to use for the workload cluster |
-|`--controlplane-machine-count`| `${CONTROL_PLANE_MACHINE_COUNT}` | The number of control plane machines to be added to the workload cluster |
+|`--controlplane-machine-count`| `${CONTROL_PLANE_MACHINE_COUNT}` | The number of control plane machines to be added to the workload cluster. See ** warning below. |
 |`--worker-machine-count`| `${WORKER_MACHINE_COUNT}` | The number of worker machines to be added to the workload cluster |
 
 Additionally, the value of the command argument to `clusterctl generate cluster <cluster-name>` (`<cluster-name>` in this case), will
 be applied to every occurrence of the `${ CLUSTER_NAME }` variable.
 
+<aside class="note warning">
+
+<h1> Warning! </h1>
+
+** `--controlplane-machine-count` today allows a value of 1 which is [not recommended for production](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#single-node-etcd-cluster) and in the future [being considered to not be supported with clusterctl code](https://github.com/kubernetes-sigs/cluster-api/issues/3425).  Instead it is recommended to [specify a value of 5](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#multi-node-etcd-cluster) or only utilize a control plane from a provider.
```

Comment:
Thanks for taking an interest in this @dtzar! However, having a `controlplane-machine-count` of 1 isn't the same as what's outlined in #3425 - that's about dropping support for standalone control-plane machines i.e. those not managed by a control plane provider. 

There's an ongoing conversation about dropping support for this in 1.4 though. @dtzar it would be great help if you could find where in the book we might be pointing toward that use case and remove those instances to help drive that work forward.

Reply:
Thanks for the feedback.  So you're saying dropping support for deploying _any_ number of control plane machines which are not managed by a provider?  Perhaps you could link to one or more places in documentation where this is referenced?

Reply:
Ok, I provided my feedback on [migrating the staging-use-cases document ](https://github.com/kubernetes-sigs/cluster-api/issues/5705#issuecomment-1324392348) all together on your issue.